R/zchunk_L226.en_distribution_USA.R

```diff
@@ -63,165 +79,411 @@ module_gcam.usa_L226.en_distribution_USA_DISABLED <- function(command, ...) {
     L226.GlobalTechShrwt_en <- get_data(all_data, "L226.GlobalTechShrwt_en")
     L226.StubTechCoef_electd <- get_data(all_data, "L226.StubTechCoef_electd")
 
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
+
+    # silence check package notes
+    region <- supplysector <- from.year <- to.year <- output.unit <- input.unit <- price.unit <-
+      logit.exponent <- logit.type <- . <- subsector <- State <- Coal <- Natural.gas <- Distillate.fuel.oil <-
+      grid_region <- state_name <- coal_adj <- gas_adj <- liq_adju <- sector1 <- adjustment <- technology <-
+      year <- minicam.non.energy.input <- tmp <- sector2 <- trash1 <- trash2 <- input.cost <- sector.name <-
+      subsector.name <- stub.technology <- market.name <- state <- NULL
+
+
+    # global_energy_to_USA_electd - takes global energy inputs from L226.en_distribution.R
+    # and processes for use in USA electricity T&D
+    global_energy_to_USA_electd <- function(data){
+      data %>%
+        filter(region == "USA",
+               supplysector %in% gcamusa.ELECT_TD_SECTORS) %>%
+        write_to_all_states(names(data)) ->
+        datanew
```

Comment:
Just end function with `write_to_all_states` call, no need to assign to `datanew`